clang/lib/AST/CanonBounds.cpp

```diff
@@ -137,6 +179,20 @@ Lexicographic::CompareScope(const DeclContext *DC1, const DeclContext *DC2) cons
   }
 }
 
+Result
+Lexicographic::CompareDecl(const CapturedDecl *CD1,
```

Comment:
Should this function be called `CompareCapturedDecl`? Because it won't accept any other `Decl`.